reconcile/gitlab_owners.py

```diff
@@ -32,8 +34,13 @@ def __init__(self, gitlab_client, merge_request, owners, dry_run):
         self.owners = owners
         self.dry_run = dry_run
 
-        top_commit = next(self.mr.commits())
-        self.top_commit_created_at = dateparser.parse(top_commit.created_at)
+        commits = self.mr.commits()
+        if len(commits) != 0:
+            top_commit = next(self.mr.commits())
```

Comment:
now you have `commits`... no need to call `mr.commits()` again, I guess

Reply:
too early in the morning